### PR TITLE
Extra opts and stats parser

### PR DESCRIPTION
**Pull request by kwaegema**
needs some more testing

### CODE REVIEW

File: lib/vsc/zk/rsync/source.py [hunk 1]
```diff
@@ -238,39 +269,79 @@ def attempt_run(self, path, attempts=3):
...
+    def parse_output(self, output):
         """
-        Runs the rsync command with or without recursion, delete or dry-run option.
-        It uses the destination module linked with this session.
+        Parse the rsync output stats, and when verbose, print the files marked for transmission
+        """
+
+        if self.rsync_verbose:
+            outp = output.split("%s%s" % (os.linesep, os.linesep))
+            self.log.info('Verbose file list output is: %s' % outp[0])
```
Comment (stdweird) [inline lib/vsc/zk/rsync/source.py]: is the numebr of files found part of the stats? otherwsie, add it

**Comment by stdweird** [comment]
looks ok

File: lib/vsc/zk/rsync/source.py (hunk 1 above)
Comment (kwaegema) [inline lib/vsc/zk/rsync/source.py]: it is part of the stats yes, both total files in the dir as the transfered files

**Comment by kwaegema** [comment]
@stdweird tested locally with 2 paths on cephtst, working.

**Pull request merged**